deploy/test/test_cases/TEST_ID_25_helm_default_retry_successful.sh

```diff
@@ -33,13 +33,16 @@ pod_name=$($cli_with_timeout get pods --namespace=$APP_NAMESPACE_NAME --selector
 
 # Find initial authentication error that should trigger the retry
 $cli_with_timeout "logs $pod_name | grep 'CSPFK010E Failed to authenticate'"
+
+sleep 1
 # Start the timer for retry interval
 start=$SECONDS
 
 echo "Expecting Secrets Provider retry configurations to take defaults RETRY_INTERVAL_SEC $DEFAULT_RETRY_INTERVAL_SEC and RETRY_COUNT_LIMIT $DEFAULT_RETRY_COUNT_LIMIT"
 $cli_with_timeout "logs $pod_name | grep 'CSPFK010I Updating Kubernetes Secrets: 1 retries out of $DEFAULT_RETRY_COUNT_LIMIT'"
```

Comment:
cli_without_timeout

Reply:
in this case i don't think it matters particularly because even if the timeout occurred the test would still fail on timing